src/transaction-types.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Temporal solution for linting error with `Buffer`

`
Unexpected use of 'Buffer'. This global is not available in the Node.js and browser environment.eslint [no-restricted-globals]
`

https://eslint.org/docs/rules/no-restricted-globals

Reply:
It'd be very nice to use a typed array here instead, but we can make that change later on. That would be a breaking change to every keyring, just for that difference.

Reply:
I would strongly prefer not using `Buffer`, as it will make this library not compatible with browsers out of the box.

Reply:
If we agree on not using `Buffer` we can just replace `bigint | string | number | Buffer` for `Bytes`